i18n/en.json

```diff
@@ -2842,6 +2842,10 @@
     "id": "api.user.complete_switch_with_oauth.blank_email.app_error",
     "translation": "Blank email"
   },
+  {
+    "id": "oauth.gitlab.tos.error",
+    "translation": "GitLab's Terms of Service have updated. Please go to gitlab.com to accept them and then try logging into Mattermost again."
```

Comment:
@jasonblais can you review the text?

Reply:
>try log in to

Isn't yours grammatically incorrect?

Reply:
If it is, then just keep yours, I was 0/5